tests/test_devices.py

```diff
@@ -60,6 +60,36 @@ def test_device_constructor_with_config(client):
     assert device.inputs_wait == {}
 
 
+def test_item_query_without_status(alarm_device):
```

Comment:
These tests are good, but to increase the robustness of the code, we should replicate and do all of them for:
- `q.INPUTS`
- `q.ALERTS`

I would say, create a class for each "query" and replicate all of them, to be sure all queries are covered and not just `q.SECTORS`.